spyder/config/main.py

```diff
@@ -1,729 +1,523 @@
+              'ipython_console/new tab': "Ctrl+T",
+              'ipython_console/reset namespace': "Ctrl+Alt+R",
+              'ipython_console/restart kernel': "Ctrl+.",
+              # ---- In widgets/arraybuider.py ----
+              'array_builder/enter array inline': "Ctrl+Alt+M",
+              'array_builder/enter array table': "Ctrl+M",
+              # ---- In widgets/variableexplorer/aarayeditor.py ----
+              'variable_explorer/copy': 'Ctrl+C',
+              # ---- In widgets/plots/figurebrowser.py ----
+              'plots/copy': 'Ctrl+C',
+              'plots/previous figure': 'Ctrl+PgUp',
+              'plots/next figure': 'Ctrl+PgDown',
+              # ---- In widgets/explorer ----
+              'explorer/copy file': 'Ctrl+C',
+              'explorer/paste file': 'Ctrl+V',
+              'explorer/copy absolute path': 'Ctrl+Alt+C',
+              'explorer/copy relative path': 'Ctrl+Alt+Shift+C',
+              }),
+            ('appearance', APPEARANCE),
+            ('lsp-server',
+             {
+              # This option is not used with the LSP server config
+              # It is used to disable hover hints in the editor
+              'enable_hover_hints': True,
+              'code_completion': True,
+              'jedi_definition': True,
+              'jedi_definition/follow_imports': True,
+              'jedi_signature_help': True,
+              'preload_modules': PRELOAD_MDOULES,
+              'pyflakes': True,
+              'mccabe': False,
+              'pycodestyle': False,
+              'pycodestyle/filename': '',
+              'pycodestyle/exclude': '',
+              'pycodestyle/select': '',
+              'pycodestyle/ignore': '',
+              'pycodestyle/max_line_length': 79,
+              'pydocstyle': False,
+              'pydocstyle/convention': 'numpy',
+              'pydocstyle/select': '',
+              'pydocstyle/ignore': '',
+              'pydocstyle/match': '(?!test_).*\\.py',
+              'pydocstyle/match_dir': '[^\\.].*',
+              'advanced/command_launch': 'pyls',
+              'advanced/host': '127.0.0.1',
+              'advanced/port': 2087,
+              'advanced/external': False,
+              'advanced/stdio': False
+             })
+            ]
+
+
+# =============================================================================
+# Config instance
+# =============================================================================
+# IMPORTANT NOTES:
+# 1. If you want to *change* the default value of a current option, you need to
+#    do a MINOR update in config version, e.g. from 3.0.0 to 3.1.0
+# 2. If you want to *remove* options that are no longer needed in our codebase,
+#    or if you want to *rename* options, then you need to do a MAJOR update in
+#    version, e.g. from 3.0.0 to 4.0.0
+# 3. You don't need to touch this value if you're just adding a new option
+CONF_VERSION = '50.1.0'
```

Comment:
Please bump this value to `50.2.0` so that the new themes are visible in our preferences by previous users.